changes/pr4487.yaml

```diff
@@ -0,0 +1,3 @@
+
+enhancement:
+  - "Replace `logging.log_to_cloud` with `cloud.send_flow_run_logs` and allow user to toggle in config - [#4487](https://github.com/PrefectHQ/prefect/pull/4487)"
```

Comment:
Isn't this a breaking change for anyone who has set this config?

Reply:
No, because the existing config value (confusingly) is not usable by the user. It's `False` by default and if set to `True` it will break `flow.run()` calls. It's only set to `True` in the _environment_ of flow execution by us. This is the major impetus for this change.

Reply:
As far as I could tell this was only togglable on the agent and https://github.com/PrefectHQ/prefect/blob/log-to-cloud/src/prefect/agent/agent.py#L146-L149 respects the existing CLI flag as well as the config now

Reply:
Ah I see, yea I thought the agent still referenced the config value but I see that it doesn't 👍